frontend/src/features/workspace/components/DeleteFormModal/DeleteFormModal.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import { MemoryRouter } from 'react-router-dom'
+import { useDisclosure } from '@chakra-ui/react'
+import { Meta, Story } from '@storybook/react'
+
+import { FormId } from '~shared/types'
+
+import { MOCK_USER, userHandlers } from '~/mocks/msw/handlers/user'
+
+import { DeleteFormModal, DeleteFormModalProps } from './DeleteFormModal'
+
+export default {
+  title: 'Pages/WorkspacePage/DeleteFormModal',
+  component: DeleteFormModal,
+  // decorators: [(storyFn) => <MemoryRouter>{storyFn()}</MemoryRouter>],
```

Comment:
should we delete this and the `MemoryRouter` import?

Reply:
cheers, f9dfdc2